### DISCUSSION TITLE
2.8 效率

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#113] [Announcements]
# 2.8 效率

SICP Python 中文翻译

https://composingprograms.netlify.app/2/8

**Comment by zhou427** [comment]
就是数据结构的时间和空间复杂度嘛，国外叫法不一样吗

**Comment by shenQQQ** [comment]
卧槽，看湿了。
用包装函数替代注解计算调用次数，所需空间，实现规划。。
太厉害了👍

**Reply by MeJinx** [reply]
## 确实厉害

**Comment by lemonacidl** [comment]
>>> fib = count(fib)
>>> fib(19)
4181
>>> fib.call_count
10946

其中fib.call_count结果出错，参照原文与复现结果，应为13529
